beacon-chain/db/restore.go

```diff
@@ -20,6 +20,9 @@ const dbExistsYesNoPrompt = "A database file already exists in the target direct
 func Restore(cliCtx *cli.Context) error {
 	sourceFile := cliCtx.String(cmd.RestoreSourceFileFlag.Name)
 	targetDir := cliCtx.String(cmd.RestoreTargetDirFlag.Name)
+	// Sanitize user input paths
```

Comment:
I will argue that we dont need these comments but rather rename function to `path.Sanitize()`. Much cleaner

Reply:
Good idea, although it requires changes to the standard Go library :-)